src/Customer.php

```diff
@@ -20,7 +20,7 @@
  * @property-read string $email
  * @property-read string $status
  * @property-read string $customer_since
- * @property-read string $attributes
+ * @property-read array $attributes
```

Comment:
Not sure why but the last two PRs (#105 and #104) didn't trigger an error for `phpstan` but there was an error on `main`. This should fix it